src/script/common/c_content.h

```diff
@@ -77,130 +77,130 @@ extern struct EnumString es_TileAnimationType[];
 
 extern const std::array<const char *, 33> object_property_keys;
 
-void               read_content_features     (lua_State *L, ContentFeatures &f,
-                                              int index);
-void               push_content_features     (lua_State *L,
-                                              const ContentFeatures &c);
+void read_content_features (lua_State *L, ContentFeatures &f,
+		int index);
```

Comment:
Just turn those into one line if it gets this small, everywhere in this file.

However, it would be good to first get opinions from other core devs before removing those alignments.
I personally don't really care, but I guess it makes sense for consistency.